readme.md

```diff
@@ -39,6 +39,57 @@ console.log(stdout);
 //=> 'unicorns'
 ```
 
+### Using the tagged templates API
+
+#### Basic
+
+```js
+import {$} from 'execa';
+
+const {stdout} = await $`echo unicorns`;
+// const {stdout} = await $`echo ${'unicorns'}`;
+// const {stdout} = await $`echo ${['unicorns', 'rainbows']}`;
+
+console.log(stdout);
+//=> 'unicorns'
+```
+
+#### With options
+
+```js
+import {$} from 'execa';
+
+await $({stdio: 'inherit'})`echo unicorns`;
+//=> 'unicorns'
+```
+
+#### With pre-defined options
+
+```js
+import {$} from 'execa';
+
+const $$ = $({stdio: 'inherit'});
+
+await $$`echo unicorns`;
+//=> 'unicorns'
+await $$({shell: true})`echo unicorns && echo rainbows`;
+//=> 'unicorns'
+//=> 'rainbows'
+```
+
+#### Synchronous
+
+```js
+import {$} from 'execa';
+
+const {stdout} = $.sync`echo unicorns`;
+console.log(stdout);
+//=> 'unicorns'
+
+$({stdio: 'inherit'}).sync`echo rainbows`;
+//=> 'rainbows'
+```
```

Comment:
The examples here do not exactly match the ones in the types. I think we should try to make them as similar as possible to ease maintenance.

https://github.com/sindresorhus/execa/pull/510/files#diff-7aa4473ede4abd9ec099e87fec67fd57afafaf39e05d493ab4533acc38547eb8R608-R625

Reply:
Good call 👍 I think the examples provided in the readme are comprehensive and demonstrate the full API, so I copied those over to the JSDoc types: https://github.com/sindresorhus/execa/pull/510/commits/5bb310ec9a9065265be381eeeab399f2f54ff995